release/pkg/controller/release/controller.go

```diff
@@ -148,7 +160,14 @@ func releaseImages(version, operatorVersion string) []string {
 	}
 }
 
-func (r *ReleaseBuilder) Build() error {
+func (r *ReleaseController) helmChartVersion() string {
+	if r.chartVersion == "" {
```

Comment:
In OSS Calico, the helm chart version is ALWAYS the same as the Calico version. We never do this thing where we release the helm chart independently from the product version.